### DISCUSSION TITLE
Display bibliography (.bib) files in Datasette ?

### DISCUSSION THREAD

**Discussion by reyman** [simonw/datasette#1189] [Ideas]
Hi Datasette guys,

Is there some plan to generate and/or display bibliography file in datasette (.bib) ? 

This could be very useful for academics website, and not so complicated to implement i suppose?,
 because all tools to generate html from bibtex exist using pandoc pandoc-citeproc.

Best regards,

**Comment by simonw** [comment]
This is a really interesting suggestion. It could certainly make for a good Datasette plugin.

Can you expand on how this might work, and what kind of things you would want to do with it?

**Reply by reyman** [reply]
The .Bib format is a very common way to describe academic research : publications, book, proceeding.
This is a database "text" file. When you transform this file you have the possibility to apply some different style (https://citationstyles.org/).

You also have some tools to manage this type of data : jabref, zotero, mendeley. 

For a researcher that need to quickly share a bibliography with the world,
 just dropping a .bib file to generate an html page could be very cool :) 

If we could link a PDF file with the corresponding bib entry (file: specific entry), thats even better ! 

There are some existing tools that do that, but few are open source : 
- https://dblp.uni-trier.de/
- https://bibbase.org/
- https://zbib.org/
- https://www.irisa.fr/lagadic/soft/bib2html/bib2html.html
- https://www.spinellis.gr/sw/textproc/bib2xhtml/

You also could use a wrapper for the universal converter Pandoc : 
- https://pypi.org/project/pypandoc/
- https://pandoc.org/demos.html

**Comment by simonw** [comment]
https://bibtexparser.readthedocs.io/en/master/ looks relevant
